### DISCUSSION TITLE
wiki.js v3 actual release date is?

### DISCUSSION THREAD

**Discussion by jdarrelljr** [requarks/wiki#4652] [General]
With today being November 1, I checked in with the wiki.js website to see if v3 has an actual release date. Unfortunately I do not find one. So my question is, What is the release date for v3.0?

As we are in the middle of planning for 2022, it would be great if someone could respond.

Thank You!
Jack

**Comment by sledgemasterrrr** [comment]
Is there any update for V3?

**Comment by hankknight** [comment]
In 2025 still waiting for V3, is there any possible info for actual date?

**Comment by czepan** [comment]
@NGPixel

**Comment by NGPixel** [comment]
Sorry, no ETA.

**Comment by RASBR** [comment]
Hi, just re-asking the same question. Any news on the release date of either Beta or final Wiki.js V3.0?

**Reply by Rizky-Indra** [reply]
There are a lot of features on the preview that I needed, kinda waiting for this one too :)

**Comment by TheAdamsFamily25** [comment]
Re-asking in 2026 😂
